caproto/threading/client.py

```diff
@@ -1282,22 +1284,33 @@ def read(self, *args, timeout=2, **kwargs):
                 f"not respond to attempt to read channel named "
                 f"{self.name!r} within {timeout}-second timeout."
             )
+
+        # TODO shouldn't we get access to the ioid of our request?
         return self.last_reading
 
     @ensure_connected
-    def write(self, *args, wait=True, cb=None, timeout=2, use_notify=True,
-              **kwargs):
+    def write(self, *args, wait=True, callback=None, timeout=2,
+              use_notify=True, **kwargs):
         "Write a new value and await confirmation from the server."
         # TODO: change use_notify default value; may break tests
         ioid = self.circuit_manager._ioid_counter()
         command = self.channel.write(*args, ioid=ioid,
                                      use_notify=use_notify,
                                      **kwargs)
+        if not use_notify:
+            # TODO:
+            # if wait or callback is not None:
+            #      raise ValueError('Not guaranteed a WriteNotifyResponse')
```

Comment:
Un-commenting this seems correct. There is no `WriteResponse`.](https://github.com/NSLS-II/caproto/blob/master/caproto/_commands.py#L1111) We should not expect any response.